File: program_data/JavaProgramData/75/735.java
import java.util.*;

package <missing>;

public class GlobalMembers
{
	/*
	 *???: party.cpp
	 *??: ???
	 *????: 2012-11-09
	 *??: 
	 */



	public static int Main()
	{
		int[] times = new int[1000];
		int[] start = new int[1000];
		int[] end = new int[1000];
		int nPeople = 0;
		int mostPeople;
		String str = new String(new char[10000]);
		str = new Scanner(System.in).nextLine();
		int readnum = 0;
		for (int i = 0; ;i++)
		{
			switch (str.charAt(i))
			{
				case ',':
				case '\0':
					start[nPeople++] = readnum;
					readnum = 0;
					break;
				default:
					readnum = readnum * 10 + str.charAt(i) - '0';
			}
			if (!str.charAt(i))
			{
				break;
			}
		}
		for (int i = 0;i < nPeople;i++)
		{
			char endchr;
			if (i < nPeople-1)
			{
				endchr = ',';
			}
			else
			{
				endchr = '\n';
			}
			cin.getline(str,(Character.SIZE / Byte.SIZE),endchr);
			end[i] = Integer.parseInt(str);
		}
		for (int i = 0;i < nPeople;i++)
		{
			for (int j = start[i];j < end[i];j++)
			{
				times[j]++;
			}
		}
		mostPeople = times[0];
		for (int i = 1;i < 1000;i++)
		{
			if (times[i] > mostPeople)
			{
				mostPeople = times[i];
			}
		}
		System.out.print(nPeople);
		System.out.print(' ');
		System.out.print(mostPeople);
		System.out.print("\n");
	}



}

File: program_data/ProgramData/104/735.cpp
int main()
{
	int A,B,i,j,a[20],b[20],n1,n2,sign=0;  //?????????A?B???????i?j????????a?b??????????n1?n2??????????sign?????
	cin>>A>>B;   //??????
	for(i=1;A>1;i++)   //??A???
	{
		a[i]=A;    //?????A??
		if(A%2==0)   //??A??????????????A/2??????A-1)/2
			A=A/2;
		else
			A=(A-1)/2;
	}
	a[i]=A;          //????????1
	n1=i;    //???????
	for(i=1;B>1;i++)   //??B????????
		{
			b[i]=B;
			if(B%2==0)
				B=B/2;
			else
				B=(B-1)/2;
		}
	b[i]=B;
	n2=i;
	for(i=1;i<=n1&&sign==0;i++)   //?????????????????
		for(j=1;j<=n2&&sign==0;j++)
			if(b[j]==a[i])
				sign=1;   //????????????1?????
		cout<<a[i-1];   //????
	return 0;
}